pkg/yang/yang.go

```diff
@@ -11,6 +11,8 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
+// Copyright 2025 Swisscom (Schweiz) AG
```

Comment:
Same comment here re: copyright statements as the other file. (No problem noting the copyright, but we should just make it compliant with usual styles.)